loops/csharp/code.cs

```diff
@@ -0,0 +1,10 @@
+﻿var u = int.Parse(args[0]); // Get an input number from the command line
+var r = new Random().Next(10000); // Get a random number 0 <= r < 10k
+var a = new int[10000]; // Array of 10k elements initialized to 0
+for (var i = 0; i < 10000; i++) { // 10k outer loop iterations
```

Comment:
Assign `a` to a new variable of type `Span<int> span` and then mutate the span, its much faster.